src/Minsk.Generators/SyntaxNodeGetChildrenGenerator.cs

```diff
@@ -31,65 +32,49 @@ public void Execute(SourceGeneratorContext context)
             var types = GetAllTypes(compilation.Assembly);
             var syntaxNodeTypes = types.Where(t => !t.IsAbstract && IsPartial(t) && IsDerivedFrom(t, syntaxNodeType));
 
+            string indentString = "    ";
```

Comment:
This variable should be const

Reply:
Agreed, 
but I also don't like the string being there in the first place.

I think that I'm going to push an update that does away with that entirely in the consuming code and moves it to the `CurlyIndenter` class.

Everything is complicated by the fact that I'm aware of @terrajobst  aversion to tabs. But the definitive argument for tabs is, for me, the one around accessibility.

Anyway, stay tuned, an update coming soon.